README.md

```diff
@@ -27,6 +27,10 @@ At the same time, a diverse and creative community of tools and services has spr
 
 Imagine a declarative Kubernetes-style API for _anything_, supported by an ecosystem of Kubernetes-aware tooling, separate from Kubernetes-the-container-orchestrator.
 
+But a control plane without any workloads isn't very compelling - so we want to uplevel the majority of Kubernetes applications today by allowing them to move and spread across clusters without changing a line of YAML. We believe moving a workload between clusters should be as easy as moving a pod between nodes.
+
+Take a multi-tenant control plane for everything, and add to it most of the world's cloud-native workloads happily cruising across clouds, datacenters, and out to the edge?
```

Comment:
nit: Maybe use exclamation instead of question mark for this sentence? Or maybe add a following sentence emphasizing how awesome the preceding vision is?

Reply:
Actually, an ellipsis would probably be the best continuation.

Reply:
Done